source/constructors/holdings.js

```diff
@@ -10,7 +10,12 @@ Pride.Core.Holdings = function(data) {
 
   const getResourceAccess = function(data) {
     const dataField = data.fields.find((field) => field.uid === 'resource_access');
-    return dataField.value;
+    if (dataField && dataField.value) {
```

Comment:
Not every record has a `resource_access` field, so dataField can be null.  This change fixes that.